### ISSUE TITLE
Video.min.js in v5.10.5 distribution is not properly minified (contains comments)

### ISSUE THREAD

**Issue by shaharmor** [videojs/video.js#3381]
## Description
The minified library has tons of docs/comments inside of it, just making the size bigger. I propose to remove all comments on minification.

## Steps to reproduce
Explain in detail the exact steps necessary to reproduce the issue.

1. Check the /dist/videojs.min.js

### Expected
No comments

### Actual
Many comments

#### videojs
what version of videojs does this occur with? 5.10.x

**Comment by OwenEdwards** [comment]
I'm looking at [video.min.js v5.10.x on the CDN](http://vjs.zencdn.net/5.10/video.min.js), and I only see **three** comments - the version & license info at the top, the `//# sourceMappingURL=video.min.js.map` at the end, and (oddly) this one in the middle:

````
/*! Time Tracking -------------------------------------------------------------- */
````

Can you point to examples of other of these "Many comments"?

**Comment by shaharmor** [comment]
I was talking about the minified files in the zip/tar in the releases page.

But if the CDN version has them removed that's good enough for me

**Comment by OwenEdwards** [comment]
@shaharmor you're right - between v5.10.4 and v5.10.5, the minified video.min.js in the ZIP on the release page has blown up! It's full of comments and nearly doubled in size. The one on the CDN is v5.10.2, so it doesn't show up.

I'd suggest you use v5.10.4 for now (v5.10.5 is prerelease anyway), and hopefully @gkatsev can dig into what broke between the two releases.

**Comment by gkatsev** [comment]
That's really weird. It's not minifying locally for me for some reason right now.

**Comment by gkatsev** [comment]
Re-installing my node_modules fixes it. Not sure what the issue was exactly. I'll publish a v5.10.6 that fixes this.

**Comment by gkatsev** [comment]
should be fixed in v5.10.6. Closing, let me know if there are still issues and we can re-open then.
